### ISSUE TITLE
Run multiple sets of commands at in parallelize

### ISSUE THREAD

**Issue by toni-moreno** [unknwon/bra#21]
My project (https://github.com/toni-moreno/resistor)  needs compile  2 different binaries, I would like handle code changes with bra for both binaries.

But when configuring with this config it doens't work

```bash
$ cat .bra.toml 
[run]
init_cmds = [
  ["go", "build","-o", "./bin/resistor", "./pkg/"],
  ["go", "build","-o", "./bin/resinjector", "./pkg/udf" ],
  ["./bin/resistor", "&", "./bin/resinjector"],
]
watch_all = true
watch_dirs = [
	"$WORKDIR/pkg",
]
```
There is any way to do this?

**Comment by unknwon** [comment]
Good idea... but currently implementation only works based on sequential execution, we may need to introduce a new key (e.g. `init_cmds_sets`) to define more than one set of commands that are executed concurrently.
